ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/ViewScanConfigDetail.tsx

```diff
@@ -160,19 +172,65 @@ function ViewScanConfigDetail({
                     )
                 )}
                 {!isLoading && scanConfig && (
-                    <Grid hasGutter>
-                        <GridItem sm={12} md={6}>
-                            <ScanConfigParameterView scanConfig={scanConfig} />
-                        </GridItem>
-                        <GridItem sm={12} md={6}>
-                            <ScanConfigProfiles profiles={scanConfig.scanConfig.profiles} />
-                        </GridItem>
-                        <GridItem sm={12}>
-                            <ScanConfigClustersTable
-                                clusterScanStatuses={scanConfig.clusterStatus}
-                            />
-                        </GridItem>
-                    </Grid>
+                    <Card>
+                        <CardBody>
+                            <Flex
+                                direction={{ default: 'column' }}
+                                spaceItems={{ default: 'spaceItemsLg' }}
+                            >
+                                <ScanConfigParametersView
+                                    headingLevel={headingLevel}
+                                    scanName={scanConfig.scanName}
+                                    description={scanConfig.scanConfig.description}
+                                    scanSchedule={scanConfig.scanConfig.scanSchedule}
+                                >
+                                    <DescriptionListGroup>
+                                        <DescriptionListTerm>Last run</DescriptionListTerm>
+                                        <DescriptionListDescription>
+                                            {scanConfig.lastExecutedTime ? (
+                                                <Timestamp
+                                                    date={new Date(scanConfig.lastExecutedTime)}
+                                                    dateFormat="short"
+                                                    timeFormat="long"
+                                                    className="pf-v5-u-color-100 pf-v5-u-font-size-md"
+                                                />
+                                            ) : (
+                                                'Scan is in progress'
+                                            )}
+                                        </DescriptionListDescription>
+                                    </DescriptionListGroup>
+                                    <DescriptionListGroup>
+                                        <DescriptionListTerm>Last updated</DescriptionListTerm>
+                                        <DescriptionListDescription>
+                                            <Timestamp
+                                                date={new Date(scanConfig.lastUpdatedTime)}
+                                                dateFormat="short"
```

Comment:
So is "Last updated" referring to when the scan configuration itself was last modified? Not going to lie the two fields (Last updated/Last run) made me scratch my head for a moment